www/app/App.js

```diff
@@ -1,54 +1,56 @@
-import React from 'react';
-import NextApp from 'next/app';
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
 import Head from 'next/head';
 import KeyboardOnlyOutlines from '@moxy/react-keyboard-only-outlines';
 import { withNextIntlSetup } from '@moxy/next-intl';
 import nextIntlConfig from '../../intl';
-import registerGoogleTracking from './ga-tracking';
+import { trackOnRouteChanged } from '../shared/utils';
 import favicon from '../shared/media/favicons/favicon.ico';
 import SEO_DATA from './App.data.js';
 
-export class App extends NextApp {
-    componentDidMount() {
-        this.unregisterGoogleTracking = registerGoogleTracking(this.props.router);
-    }
+const App = ({ Component, pageProps, router }) => {
```

Comment:
```suggestion
export const App = ({ Component, pageProps, router }) => {
```

Tests are failing because we are not exporting the app here.